docs/managing-dragonfly/clustermode.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 5
+---
+
+# Cluster Mode
+
+Dragonfly can reach the capacity of a multi node Redis Cluster on a single instance.
+In order to help with migrating an application from a Redis Cluster to Dragonfly, Dragonfly can emulate a Redis Cluster.
+
+```bash
+# Running Dragonfly instance in cluster mode
+$ dragonfly --cluster_mode=emulated
+$ redis-cli 
+# See which cluster commands are supported
+127.0.0.1:6379> cluster help
+```
+
```

Comment:
I would also specify a two-liner in python/js on how a cluster-enabled client can connect  to Dragonfly.

Reply:
Not sure what mean, there are many clients out there with different configuration, some will work with dragonfly even without the --cluster-mode flag just by the virtue of specifying a single target host. 

Deepening on the tech depth of the user on the behaviour of their specific client it may be hard to know in advance if they should use the flag other than trial and error.

Of course there's no harm with always specifying the flag, that is why I suggested to drop it completely.